data/hai/hai missions.txt

```diff
@@ -1028,6 +1030,7 @@ mission "Hiding in Plain Sight"
 	on complete
 		payment 75000
 		dialog `Arthur and Kiru pay you <payment> before running off to the convention center. As they run away, you notice that Kiru is drawing a number of eyes from the crowd, but no one seems to suspect that he is an actual alien.`
+		log "People" "Arthur and Kiru" `Human and Hai respectively, both attended the Vinci cosplay convention without a costume, gaining much attention.`
```

Comment:
Given that these likely won't be repeat characters, I'm not so certain they should have a log entry.